### PR TITLE
Tried to make a couple things clearer

### PR DESCRIPTION
**Pull request by diogeneshamilton**
@marklarr great README! I tried to improve a couple things, let me know if you agree it makes things a bit clearer.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -43,13 +43,13 @@ $ slather setup path/to/project.xcodeproj
 This will enable the `Generate Test Coverage` and `Instrument Program Flow` flags for your project.
 
 
-To test if you're ready to generate test coverage, run your test suite on your project, and then run:
+To verify if you're ready to generate test coverage, run your test suite on your project, and then run:
```
Comment (kylef) [inline README.md]: When `test` is changed to `verify`, `if` no longer fits here.

I'd suggest removing that too:

```diff
- To verify if you're ready...
+ To verify you're ready...
```

File: README.md [hunk 2]
```diff
@@ -122,14 +122,14 @@ $ slather coverage -x --output-directory path/to/xml_report
 ### Coverage for code included via CocoaPods
 
 If you're trying to compute the coverage of code that has been included via
-CocoaPods, you will need to tell CocoaPods to use the slather plugin by
+CocoaPods, you will need to tell CocoaPods to use the Slather plugin by
```
Comment (kylef) [inline README.md]: The plugin name is actually `slather` and not `Slather`, it is case-sensitive.

Comment (diogeneshamilton) [inline README.md]: Yes, but in this case it's being used in a sentence as a proper noun, so it's capitalized. The sample code is correctly lowercase.

File: README.md (hunk 1 above)
Comment (diogeneshamilton) [inline README.md]: Sounds good! I will change.

**Comment by marklarr** [comment]
thanks @diogeneshamilton

**Pull request merged**